packages/modules/src/utils/Constants.ts

```diff
@@ -20,17 +20,17 @@ export const ECDSA_OWNERSHIP_MODULE_ADDRESSES_BY_VERSION = {
   V1_0_0: '0x0000001c5b32F37F5beA87BDD5374eB2aC54eA8e'
 }
 
-export const DEFAULT_SESSION_KEY_MANAGER_MODULE = '0x000002FbFfedd9B33F4E7156F2DE8D48945E7489'
+export const DEFAULT_SESSION_KEY_MANAGER_MODULE = '0x6E49e404BD70bcc4756F1057d2E2e6000cD38e1e'
```

Comment:
This address is not starting with initial 0s like other addresses?

Reply:
yes answered in our slack chat